app/helpers/blacklight/facets_helper_behavior.rb

```diff
@@ -124,7 +124,7 @@ def facet_partial_name(display_facet = nil)
   # @return [String]
   def render_facet_value(facet_field, item, options ={})
     path = path_for_facet(facet_field, item)
-    content_tag(:span, :class => "facet-label") do
+    content_tag(:span, :class => "facet-label col-xs-10 col-sm-8 col-md-9 col-lg-10") do
```

Comment:
Can we pull this list out into a helper or something (@extend in the CSS for `.facet-label`?) to avoid repeating it?